src/main/resources/liquibase/crop_changelog/21_4_0.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="lkovacic" id="v21.4.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<indexExists tableName="methods" indexName="methods_mcode_uindex"/>
```

Comment:
@lik-83 can we add a preCondition to check that there are no duplucated values?

Reply:
Done!